repose-aggregator/installation/deb/pom.xml

```diff
@@ -23,6 +23,43 @@
         <module>repose-valve</module>
     </modules>
 
+    <profiles>
+        <profile>
+            <!-- this profile works with gnupg 2.0.x NOT with 2.1.x -->
+            <id>release-sign-artifacts</id>
```

Comment:
Yes. This is the id used in the top level to extract the environment variables into maven properties and in the DEB and RPM POM's to indicate to actually use them.